z3alpha/multiprocess_evaluator.py

```diff
@@ -114,7 +114,7 @@ def run_solver(
         runtime = time_after - time_before
         
         lines = out.decode("utf-8").split("\n")
-        res = lines[0] if lines else "error"
+        res = lines[0] if lines and len(lines[0]) > 0 else "error"
```

Comment:
Just curious, did you encounter cases where `lines` is true but lines[0] is an empty string?